manifest.json

```diff
@@ -17,19 +17,25 @@
   "name": "Web Thing",
   "options": {
     "default": {
-      "urls": []
+      "urls": [],
+      "poolInterval": 5000
     },
     "schema": {
       "type": "object",
       "required": [
-        "urls"
+        "urls",
+        "poolInterval"
       ],
       "properties": {
         "urls": {
           "type": "array",
           "items": {
             "type": "string"
           }
+        },
+        "poolInterval": {
+          "description": "The interval, in milliseconds, at which to poll property values.",
```

Comment:
I changed to milliseconds in order to avoid *1000 in code. If seconds is preferable let me know.